LondonBritishLibrary/orient/BLorient2263.xml

```diff
@@ -0,0 +1,219 @@
+                                ስብሐት፡ ለኪ፡ ቤተ፡ ክርስቲያን፡ ዘአብሐኪ፡ እግዚአብሔር።
+                                በላዕለ፡ ሥጋሁ፡ ቅዱስ፡ ወደሙ፡ ክቡር።</explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="6ra" to="12vb"/>
+                            <title>Turkish and Arabic vocabulary</title>
+                            <note>As a whole indicated by: <foreign xml:id="ti"><title>ነገረ፡ ትርኵ። ዓ።</title></foreign> written on the upper
+                                margin.</note>
+                        <msItem xml:id="ms_i3.1">
+                            <locus target="#6ra"/>
+                            <title ref="NAR0184ForeignWords">Tigrinya-Turkish Vocabulary</title>
+                            <note>Containing only eight words.</note>
+                            <incipit xml:lang="ti">ነዓ፡ ብሂል፡ ዢል፤
+                                ተቀመጥ፡ ብ፡ ውቱር፤
+                                ሀባ፡ ብ፡ ዠትር፤
+                                አልቦን፡ ብ፡ ያክትር፤
+                                ሀሎ፡ ብ፡ ዮል፤
+                                ኪድ፡ ብ፡ ቢናዓደም፤
+                                መጽአ፡ ብ፡ ገልድ፤
+                                ኬደ፡ ብ፡ ደልዳል።</incipit>
+                            <textLang xml:lang="ti"/>
+                            </msItem>
+                            <msItem xml:id="ms_i3.2">
+                                <locus from="6ra" to="12vb"/>
+                                <title ref="NAR0184ForeignWords">Tigrinya-Arabic Vocabulary</title>
+                                <note>Containing several words on <locus target="6ra"/> with a special ancient form of the letter 
+                                    <foreign xml:lang="ti">ዥ</foreign> with small strokes in every of the four edges.</note>
+                                <incipit xml:lang="ti">ነዓ፡ ብሂል፡ ተአል፤
+                                    ተቀመጥ፡ ብ፡ አይዥልስ፤
+                                    ተንሥእ፡ ብ፡ ጉም፡ ቁምሂ፤
+                                    ኪድ፡ ብ፡ ሩሕ፤
+                                    ኬድኩ፡ ብ፡ ሮሕ፡ 
+                                    ኬደ፡ ብ፡ ራሕ፡ መሻ፤
+                                    ተመልሰ፡ ብ፡ ገለበ፤
+                                </incipit>
+                                <textLang xml:lang="ti"/>
+                            </msItem>
+                        </msItem> 
+                        <msItem xml:id="ms_i4">
+                            <locus from="13ra" to="15va"/>
+                            <title ref="LIT7031HymnJesus" type="incomplete">Hymn to Jesus Christ</title>
+                            <note>The beginning is missing.</note>
+                            <incipit xml:lang="gez">አብ፡ ቅዱስ፡ አምጻኤ፡ ዓለማት፡ በኪኑ።
+                                ወልድ፡ ቅዱስ፡ በምኵናኑ።
+                                መንፈስ፡ ቅዱስ፡ ዘኢይትሌለይ፡ ሥልጣኑ።
+                                ለሥላሴሆሙ፡ ዘኢይትረከብ፡ ወሰኑ።
+                                ወህላ<add resp="PRS8999Strelcyn">ዌ</add>ሆሙ፡ ተከሥተ፡ ለመኑ።
```

Comment:
I would have used 'supplied reason=omitted' for this kind of editorial interventions. What do you think?

Reply:
You are right - add is the incorrect markup in that place.